qa/tasks/cephfs/test_layout.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+from tasks.cephfs.cephfs_test_case import CephFSTestCase
+
+log = logging.getLogger(__name__)
+
+class TestInheritedLayout(CephFSTestCase):
+
+    def test_layout_root(self):
+        self.mount_a.run_shell(["mkdir", "a"])
+        layout = self.mount_a.getfattr("a", "ceph.dir.layout") 
+        log.info(layout)
+
+    def test_layout(self):
+        self.mount_a.run_shell(["mkdir", "-p", "a/b"])
+        self.mount_a.setfattr("a", "ceph.dir.layout.stripe_count", "2")
+        layout_a = self.mount_a.getfattr("a", "ceph.dir.layout")
+        log.info(layout_a)
+        layout_b = self.mount_a.getfattr("b", "ceph.dir.layout")
```

Comment:
now compare?

Test name could be improved. Please add a doc string indicating the purpose of each test.

I also think we could move this class to `test_misc.py`.

Reply:
Ack. 

Yea regex and compare the layouts and then check if inherit=true is also a part of layout_b seems right. I'll update.

Setting explicit layout, clearing the layout and then checking if layout is inherited is another test that I can think of. Any other one's come to your mind?